gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/statistics/functions/ExtractPropertyValuesByLabel.java

```diff
@@ -27,8 +27,9 @@
 import java.util.Set;
 
 /**
- * Extracts Triples of the form <Tuple2<Label,PropertyName>, PropertyValue> from the given list of
+ * Extracts Triples of the form {@code <Tuple2<Label,PropertyName>, PropertyValue>} from the given list of
```

Comment:
That would be (in pseudocode, not with actual types): `Tuple2<Tuple2<Label, PropertyName>, PropertyValue>`